src/AdView.tsx

```diff
@@ -67,77 +74,45 @@ const AdViewComponent = requireNativeComponent<AdViewProps & ViewProps & AdViewN
 
 type AdViewType = React.Component<AdViewProps> & NativeMethods;
 
+type SizeKey = 'width' | 'height';
+type SizeRecord = Partial<Record<SizeKey, number | string | null>>;
```

Comment:
For banners/MRECs, how can the value be anything but an int? Thus isn't `number | string | null` overkill?

Reply:
`null` is removed since it made nonull.   Width/height can be a string like '100%' and 'auto'.